problem_builder/templates/html/swipeblock.html

```diff
@@ -0,0 +1,30 @@
+{% load i18n %}
+{% if not hide_header %}
+<h4 class="question-title" id="heading_{{ self.html_id }}">{{ self.display_name_with_default }}</h4>
+{% endif %}
+<fieldset class="choices questionnaire" id="{{ self.html_id }}">
+  <legend class="question field-group-hd">{{ self.question|safe }}</legend>
```

Comment:
Nit: It would be good to include an `<img>` element with the `src` attribute set to the expanded `img_url`.